libs/ui/inputs/src/lib/switch-toggle/switch-toggle.component.ts

```diff
@@ -10,7 +10,7 @@ import { MatButtonToggleModule } from '@angular/material/button-toggle'
 
 export type SwitchToggleOption = {
   label: string
-  value: string
+  value: any
```

Comment:
Is there a way we could still type this?

Reply:
I removed it completely since it will be of different value for each component that will use the SwitchToggleComponent. And it is also not used anymore inside the component. WDYT?